CHANGES.md

```diff
@@ -51,24 +51,24 @@ replace with next unreleased version
 ### BUG FIXES
 
 - List bugs fixed here.
```

Comment:
I'm now recommending we use this new approach of using the jira-generated release notes to build the changelog.